src/main/java/duke/command/DeleteCommand.java

```diff
@@ -0,0 +1,38 @@
+package duke.command;
+
+import duke.Storage;
+import duke.TaskManager;
+import duke.exception.DukeException;
+import duke.task.Task;
+
+public class DeleteCommand extends Command {
+    private int taskIndex;
+
+    /**
+     *  DeleteCommand constructor.
+     *
+     *  @param index Index of task to be deleted.
+     */
+    public DeleteCommand(int index) {
```

Comment:
Could have considered the param to be named as taskIndex also to match the class attribute